Cargo.toml

```diff
@@ -2,7 +2,7 @@
 members = ["fuzz"]
 
 [workspace.package]
-version = "0.0.8+llvm-462a31f5a5ab"
+version = "0.1.0+llvm-462a31f5a5ab"
```

Comment:
I'm not sure what the convention is for this kind of thing but should we consider going to `0.2` since crates.io already has a `0.1` series? (Might be clearer that way to anyone currently using that crate)